owncloudComLibrary/src/test/java/com/owncloud/android/lib/ShareeResponseTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.owncloud.android.lib
```

Comment:
From my point of view, tests should be included in the same package as the original but in the test folder, i mean, if `ShareeResponse` is inside package `com.owncloud.android.lib.resources.shares.responses`, this test should be in the same hierarchy and not in the root folder.